client/bin/multi-client.js

```diff
@@ -38,10 +39,51 @@ const yargs = require('yargs')
 		type: 'boolean',
 		default: false,
 	})
+	.option('n', {
+		alias: 'none-interactive',
```

Comment:
NIT: non-interactive not none-interactive

Reply:
thanks